media_driver/agnostic/gen12/codec/hal/codechal_decode_scalability_g12.cpp

```diff
@@ -668,6 +668,7 @@ MOS_STATUS CodecHalDecodeScalability_FEBESync_G12(
     pOsInterface = pScalabilityStateBase->pHwInterface->GetOsInterface();
     pMiInterface = pScalabilityState->pHwInterface->GetMiInterface();
     HcpDecPhase = pScalabilityStateBase->HcpDecPhase;
+    curPipeInUse = 0;
```

Comment:
why not initialize variable when define?  such as line 655 to "uint32_t curPipeInUse = 0;"

Reply:
yes, it should be initialized at the moment of definition. I have modified again.